.vscode/settings.json

```diff
@@ -6,5 +6,8 @@
   "editor.formatOnSave": true,
   "editor.defaultFormatter": "esbenp.prettier-vscode",
   "prettier.prettierPath": "./node_modules/prettier",
+  "files.associations": {
+    "base.json": "jsonc"
+  },
```

Comment:
Because the `base.json` file isn't named `tsconfig.*.json`, VS Code doesn't provide correct syntax highlighting and autocomplete by default. This and the `$schema` provide workarounds.